.github/workflows/reindex-website-main.yml

```diff
@@ -15,3 +15,27 @@ jobs:
           username: ${{ secrets.ALGOLIA_CRAWLER_USER_ID }}
           password: ${{ secrets.ALGOLIA_CRAWLER_API_KEY }}
           customHeaders: '{"Content-Type": "application/json"}'
+  check-index:
+    runs-on: ubuntu-latest
+    name: Checks that the latest index is still working
+    steps:
+      - uses: fjogeleit/http-request-action@v1
+        id: search
+        with:
+          url: "https://NTSX40VZB8-dsn.algolia.net/1/indexes/kurtosis/query"
+          method: 'POST'
+          customHeaders: '{"X-Algolia-API-Key": "4269c726c2fea4e6cddfeb9a21cd3d4e", "X-Algolia-Application-Id": "NTSX40VZB8"}'
+          data: '{ "params": "query=add_service&hitsPerPage=2&getRankingInfo=1" }'
+      - name: Show Response
+        run: |
+          #!/bin/bash
+          set -eou pipefail
+
+          if (( ${{ fromJson(steps.search.outputs.response).nbHits }} > 100 )); then
```

Comment:
again, not familiar with Algolia API, but more than 100 hits for `add_service` query? Is that because of all the different versions we have or something else?

Reply:
Yep there's about 600 of them thanks to the different versions. Can adjust to something lower if this starts failing